Packs/TaniumThreatResponse/Integrations/TaniumThreatResponse/TaniumThreatResponse.yml

```diff
@@ -1683,6 +1718,74 @@ script:
     - contextPath: Tanium.DownloadFile.Downloaded
       description: The date of the download request.
       type: Date
+  - arguments:
+    - description: Intel Document in OpenIOC format containing hashes
+      name: intel-doc
+      required: true
+    description: Upload intel document to Tanium Threat Response
+    name: tanium-tr-intel-doc-create
+    outputs:
+    - contextPath: Tanium.IntelDoc.AlertCount
+      description: The number of alerts that currently exist for this intel.
+      type: number
+    - contextPath: Tanium.IntelDoc.CreatedAt
+      description: The date at which this intel was first added to the system.
+      type: date
+    - contextPath: Tanium.IntelDoc.Description
+      description: The description of the intel, as declared in the document or as
+        updated by a user.
+      type: string
+    - contextPath: Tanium.IntelDoc.ID
+      description: The unique identifier for this intel in this instance of the system.
+      type: number
+    - contextPath: Tanium.IntelDoc.LabelIds
+      description: The IDs of all labels applied to this intel.
+      type: number
+    - contextPath: Tanium.IntelDoc.Name
+      description: The name of the intel, as declared in the document or as updated
+        by a user.
+      type: string
+    - contextPath: Tanium.IntelDoc.UnresolvedAlertCount
+      description: The number of unresolved alerts that currently exist for this intel.
+      type: number
+    - contextPath: Tanium.IntelDoc.UpdatedAt
+      description: The date when this intel was last updated.
+      type: date
+  - arguments:
+    - description: The intel document ID.
+      name: intel_doc_id
+      required: true
+    - description: The name of a Tanium computer group. See command description for
+        possible ways to retrieve this value.
+      name: computer_group_name
+      required: true
+    description: Scan a computer group for hashes in intel document. Computer groups
+      can be viewed by navigating to `Administration -> Computer Groups` in the UI.
+      Computer group names and IDs can also be retrieved by using the `tn-list-groups`
+      command in the `Tanium` integration.
+    name: tanium-tr-start-quick-scan
```

Comment:
@DanilVil Is this description satisfying?
Regarding what you wrote [here](https://github.com/demisto/content/pull/14811#issuecomment-941001196).